Volume/Views/BookmarksList.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  BookmarksList.swift
+//  Volume
+//
+//  Created by Cameron Russell on 11/15/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct BookmarksList: View {
+    var body: some View {
+        GeometryReader { geometry in
+            NavigationView {
+                ScrollView(showsIndicators: false) {
+                    Header(text: "Saved Articles")
+                    if let savedArticles = articleData.filter{ $0.saved }, savedArticles.count > 0 {
+                        ForEach (savedArticles) { article in
```

Comment:
Instead of putting a ForEach in a ScrollView (bad performance), can we not do List because of the hairlines?

Reply:
I briefly found a workaround to the hairlines problem and probably should have documented in the DropBox file we have going, but that slipped my mind. :( Perhaps we should get a doc going for all the things we will refactor once Apple releases a new version of SwiftUI that solved many of the current problems were facing (and the convoluted workarounds we're perhaps implementing).